src/rpft/rapidpro/models/routers.py

```diff
@@ -202,6 +202,7 @@ def generate_category_name(self, comparison_arguments):
         # Auto-generate a category name that is guaranteed to be unique
         # TODO: Write tests for this
```

Comment:
This todo comment was written over 4 years ago - I think it's time it was removed. Personally, I would only use todos when work is in progress and remove them before merging.

Reply:
Fair enough, I guess it's not going to happen (and not important enough to capture in an issue?)